config/routes.rb

```diff
@@ -84,15 +88,23 @@
       end
 
       scope path: "lead-providers" do
-        resources :lead_provider_users, only: %i[edit update], path: "users"
+        resources :lead_provider_users, only: %i[edit update destroy], path: "users" do
+          member do
+            get "delete", action: :delete
+          end
+        end
       end
     end
 
     scope :administrators, module: "administrators" do
-      resources :administrators, only: %i[index new create edit update], path: "/" do
+      resources :administrators, only: %i[index new create edit update destroy delete], path: "/" do
         collection do
           post "new/confirm", action: :confirm, as: :confirm
         end
+
+        member do
+          get "delete", action: :delete
+        end
```

Comment:
Do we need `destroy delete` in `resources` definition and the `member` delete?

Reply:
Updated